src/pages/settings/Profile/CustomStatus/SetTimePage.tsx

```diff
@@ -40,24 +43,18 @@ function SetTimePage({translate, customStatus}) {
             />
             <View style={styles.flex1}>
                 <TimePicker
-                    inputID="timePicker"
                     defaultValue={clearAfter}
-                    style={styles.flexGrow1}
```

Comment:
Why was this props removed?

Reply:
`TimePicker` does not accept `inputID` and `style` props after the migration